### PR TITLE
Storybook > Menu bar

### PR DESCRIPTION
**Pull request by edderrd**
## Acceptance criteria

1. "L" level protocol logo
2. Current time
3. Current date

### Screenshot

Screen Shot 2021-12-21 at 3 21 46 PM

### CODE REVIEW

**Comment by edderrd** [comment]
[Storybook > Menu bar](https://trello.com/c/kjkkZYBr/16-storybook-menu-bar)

**Review by gustavoalfaro — APPROVED** [review]
nice

File: components/ui/MenuBar.js [hunk 1]
```diff
@@ -81,8 +83,8 @@ const Toolbar = ({ brand, children }) => {
         <li className="brand">
           <Link href="/" passHref>
             <>
-              {brand}
-              <ChevronIcon />
+              <LevelLogoIcon />
+              Level Protocol
```
Comment (mckelvey) [inline components/ui/MenuBar.js:87]: @edderrd — Just wondering why we have the date values as links? Can we just have those as static content? (I’d taken the previous `<a>`s and converted them to next/link without thinking about this so I own it too, but I doubt we’ll make them into anything at present.)

**Review by mckelvey — CHANGES_REQUESTED** [review]
@edderrd — One quick change, now that we’re closer on the Figma. :)

File: components/ui/MenuBar.js (hunk 1 above)
Comment (mckelvey) [reply components/ui/MenuBar.js:87]: Note: this comment is above the change; it’s as close to the items as possible.

Comment (edderrd) [reply components/ui/MenuBar.js:89]: converted to tray items to avoid using those links

Comment (mckelvey) [reply components/ui/MenuBar.js:89]: That’s a good solution @edderrd — will you remove the hover on `tray-item` since it’s not an actionable item.

Comment (edderrd) [reply components/ui/MenuBar.js:96]: done and also remove active to avoid confusion when clicked

Comment (mckelvey) [reply components/ui/MenuBar.js:96]: Thanks!

**Review by mckelvey — APPROVED** [review]
👍

**Pull request merged**